doc/source/_static/js/assistant.js

```diff
@@ -1,15 +1,15 @@
 // Create chat-widget div
-var chatWidgetDiv = document.createElement("div");
-chatWidgetDiv.className = "chat-widget";
+var chatWidgetDiv = document.createElement('div');
+chatWidgetDiv.className = 'chat-widget';
```

Comment:
what are these styling changes? is is related to AI assistant breaking?

Reply:
Ugh, no. This is caused by the fact that I'm running pre-commit, but it hasn't yet been run on anything in `doc/`. I'm opening up an issue and a PR to do this now so we don't have to keep worrying about this. See https://github.com/ray-project/ray/pull/42589.